src/components/molecules/filter-dropdown/item.tsx

```diff
@@ -0,0 +1,423 @@
+          className={clsx(
+            "py-1.5 px-3 flex w-full items-center hover:bg-grey-5 rounded justify-between",
+            {
+              "inter-small-semibold": open,
+              "inter-small-regular": !open,
+            }
+          )}
+        >
+          <div className="flex items-center">
+            <div
+              className={`w-5 h-5 flex justify-center text-grey-0 border-grey-30 border rounded-base ${
+                open && "bg-violet-60"
+              }`}
+            >
+              <span className="self-center">
+                {open && <CheckIcon size={16} />}
+              </span>
+            </div>
+            <input
+              id={filterTitle}
+              className="hidden"
+              checked={open}
+              type="checkbox"
+            />
+            <span className="ml-2">{filterTitle}</span>
+          </div>
+          {open && (
+            <span className="text-grey-50 self-end">
+              <ChevronUpIcon size={20} />
+            </span>
+          )}
+        </RadixCollapsible.Trigger>
+        <RadixCollapsible.Content className="w-full">
+          {filterTitle === "Date" ? (
+            <DateFilter
+              options={options}
+              open={open}
+              setFilter={setFilter}
+              existingDate={filters}
+              filterTitle={filterTitle}
+            />
+          ) : (
+            options.map((el, i) => (
+              <div
+                className={clsx(
+                  "w-full flex hover:bg-grey-20 my-1 py-1.5 pl-6 items-center rounded",
+                  {
+                    "inter-small-semibold": checked[el],
+                    "inter-small-regular": !checked[el],
+                  }
+                )}
+                key={i}
+                onClick={() => onCheck(el)}
+              >
+                <div
+                  className={`w-5 h-5 flex justify-center text-grey-0 border-grey-30 border mr-2 rounded-base ${
+                    checked[el] === true && "bg-violet-60"
+                  }`}
+                >
+                  <span className="self-center">
+                    {checked[el] === true && <CheckIcon size={16} />}
+                  </span>
+                </div>
+                <input
+                  type="checkbox"
+                  className="hidden"
+                  id={el}
+                  name={el}
+                  value={el}
+                  checked={checked[el] === true}
+                  style={{ marginRight: "5px" }}
+                />
+                {el}
+              </div>
+            ))
+          )}
+        </RadixCollapsible.Content>
+      </RadixCollapsible.Root>
+    </div>
+  )
+}
+
+export default FilterDropdownItem
+
+const DateFilter = ({
+  options,
+  open,
+  setFilter,
+  existingDate,
+  existingFilter,
+  filterTitle,
+}) => {
+  const [currentFilter, setCurrentFilter] = useState(
+    existingFilter || undefined
+  )
```

Comment:
It seems, that the date filter does not hold the value of the applied filter. So when you have applied a filter and re-open the date filter item, the value of the applied filter is no longer present. 

![image](https://user-images.githubusercontent.com/59018053/151425788-181ac29a-deff-4661-8e0a-e8f8bf996f9c.png)

Reply:
@pKorsholm Do you have an idea, why this happens?